nmrfx-processor-gui/src/main/java/org/nmrfx/processor/gui/spectra/KeyBindings.java

```diff
@@ -128,7 +128,7 @@ public void keyPressed(KeyEvent keyEvent) {
                 case BACK_SPACE:
                     keyMonitor.complete();
                     keyEvent.consume();
-                    chart.deleteSelectedPeaks();
+                    chart.deleteSelectedItems();
```

Comment:
I think the Delete key should be deleting the selected items as well as backspace, I would normally try deleting with "delete" before attempting to use "backspace" as a delete for anything that is not in a textfield

Reply:
merged DELETE and BACK_SPACE case